packages/dds/map/src/mapKernel.ts

```diff
@@ -671,6 +683,10 @@ export class MapKernel implements IValueTypeCreator {
                     // We don't reuse the metadata but send a new one on each submit.
                     this.submitMapClearMessage(op);
                 },
+                rebaseOp: (op: IMapClearOperation, localOpMetadata: unknown) => {
+                    // We don't reuse the metadata but send a new one on each submit.
```

Comment:
I've left a comment earlier on that: I think metadata should not be here on API as original local metadata is not serializable (it contains promises, for example, in case of consensus data structures, if I remember correctly).
I believe we do not need it, as it should be recreated by DDSs when re-applying local changes